src/components/Form/spec.js

```diff
@@ -480,7 +480,7 @@ describe('Form component', () => {
       const description = component.find('.rendition-form-description').first()
 
       expect(description.html()).toMatch(
-        '<p>Lorem ipsum <em>dolor</em> sit amet</p>'
+        '<div id="root_foo__description" class="sc-fzoyAV cFKUiM sc-fzoLag jAOLUb sc-fznNTe glwMIL rendition-form-description"><div><p class="sc-fzoyAV cFKUiM sc-fzpjYC gJiVIn">Lorem ipsum <em>dolor</em> sit amet</p></div></div>'
```

Comment:
I think you'll have to re-think these tests as the class names will change 'randomly'. Can this be turned into a snapshot test?

Reply:
Good catch, I'll convert these to snapshot tests.